### PR TITLE
Switch from CircleCI to GH-Actions

### PR DESCRIPTION
**Pull request by stuxcrystal**

### CODE REVIEW

**Comment by kageru** [comment]
Is this ready for merge? I don’t know much about GH actions, but it looks fine to me.

Also, can I squash this? I’d rather not have 35 commits for CI setup in the history.

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        vs-version: [latest,50, 49]
```
Comment (kageru) [inline .github/workflows/run-tests.yml:18]: Do we really need to check compatibility with 3 different versions? Latest is fine, but someone would have to update the `50, 49` whenever a new version is released. I don’t want to do that.

**Comment by FichteFoll** [comment]
> Also, can I squash this?

Definitely, imo.

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Run tests
+
+on:
+  push:
+    branches: [ master ]
```
Comment (FichteFoll) [inline .github/workflows/run-tests.yml:8]: Why only run this for the master branch? Same for pull requests against master.

Comment (stuxcrystal) [reply .github/workflows/run-tests.yml:8]: Right now we don't have any other branch, do we?

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (stuxcrystal) [reply .github/workflows/run-tests.yml:18]: I'd rather make sure we have some policy that defines which versions each vsutil version supports. Putting it here cements this policy.

I'd like to make sure that a user doesn't have to switch to a new version immediately, as sometimes VapourSynth releases are broken. At the same time I want to have a codified place that authoritatively defines the supported versions.

**Comment by stuxcrystal** [comment]
@kageru It's ready to merge. I added commits to switch to my new github-action that installs vapoursynth (44-50 right now) for you.

File: .github/workflows/run-tests.yml (hunk 2 above)
Comment (FichteFoll) [reply .github/workflows/run-tests.yml:8]: Currently, yes, but if you were to ever develop on a different branch, even for pull requests (on this repo), the action would not run until the PR was created.

**Pull request merged**